site/layouts/partials/social.html

```diff
@@ -1,17 +1,25 @@
-{{ "<!-- Twitter -->" | safeHTML }}
-<meta name="twitter:card" content="{{ if .IsHome }}summary_large_image{{ else }}summary{{ end }}">
+<meta name="twitter:card" content="summary_large_image">
 <meta name="twitter:site" content="@{{ .Site.Params.twitter }}">
 <meta name="twitter:creator" content="@{{ .Site.Params.twitter }}">
 <meta name="twitter:title" content="{{ .Title | markdownify }}">
 <meta name="twitter:description" content="{{ .Page.Params.description | default .Site.Params.description | markdownify }}">
-<meta name="twitter:image" content="{{ if .IsHome }}{{ .Site.Params.social_logo_path | absURL }}{{ else }}{{ .Site.Params.social_image_path | absURL }}{{ end }}">
 
-{{ "<!-- Facebook -->" | safeHTML }}
+{{ if .Page.Params.thumbnail }}
+<meta name="twitter:image" content="/docs/{{ .Site.Params.docs_version }}/assets/img/{{ .Page.Params.thumbnail }}">
+{{ else }}
+<meta name="twitter:image" content="{{ .Site.Params.social_image_path | absURL }}">
```

Comment:
Since `social_image_path` is used only in this file. Maybe we could get rid of this param in `config.yml` (to avoid having something to check after running `npm run release 5.1.3 5.2.0`) in order to use here:
```
<meta name="twitter:image" content="/docs/{{ .Site.Params.docs_version }}/assets/brand/bootstrap-social.png">
```
(Or just store the filename in `config.yml` instead of the full path).

If you do that, the line could be simplified in something like (haven't tested it but you get the spirit):

```
<meta name="twitter:image" content="/docs/{{ .Site.Params.docs_version }}/assets/{{ if .Page.Params.thumbnail }}img/{{ .Page.Params.thumbnail }}{{else}}brand/bootstrap-social.png{{end}}">
```

Speaking of `config.yml`, `social_logo_path` doesn't seem to be used anymore.